dripconfig/builtins.py

```diff
@@ -82,16 +83,33 @@ class StatsdConfig(SchemaBasedTrigger):
         'statsd': {
             Required('host'): basestring,
             Required('port', default=8125): Coerce(int),
+            Optional('sample_rate', default=1): Coerce(int),
+            Optional('disabled', default=False): bool,
         },
     })
 
     def configure(self, configuration):
+        """
+        If statsd is missing from the config, the emission of
```

Comment:
Won't statsd *always* be in the config? (ConfigDict calls configure of each trigger on `self`)

Furthermore, statsd should work by default. If someone is using this builtin trigger, it's probably because they want magic 'just works' functionality. Running the statsd client in disabled mode is kind of an odd edge case and it should only take effect if set explicitly.